### PR TITLE
ci(mergify): change label merge condition

### PR DESCRIPTION
**Pull request by jd**
As every PR is now labeled `automerge`, and people are now forgetting to add
the automerge label, let's change the merge rule to always automerge except
said otherwise.

### CODE REVIEW

**Review by P403n1x87 — APPROVED** [review]
Fine by me

File: .mergify.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pull_request_rules:
   - name: automatic merge
     conditions:
-      - label=automerge
+      - label!=manual merge
       - check-success=test
       - "#approved-reviews-by>=1"
```
Comment (brettlangdon) [inline .mergify.yml:6]:
```suggestion
      - "#approved-reviews-by>=2"
```

?

Comment (brettlangdon) [reply .mergify.yml:6]: I know GitHub API will block it, but would it be good to enforce it here too?

Comment (jd) [reply .mergify.yml:6]: It's a good question!
We definitely can. I noticed there's a little advantage to keep the `1` here, which is that Mergify will start updating the PR (merge the base branch) as soon as it has 1 approved review. That keeps the PR ready to be merged as soon as the second approval is received.


I'll send a PR to do this in a more explicit way.

File: .mergify.yml [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
-      - label=automerge
+      - label!=manual merge
       - check-success=test
-      - "#approved-reviews-by>=1"
+      - "#approved-reviews-by>=2"
     actions:
       merge:
         strict: smart
         method: squash
+  # This rule keep the PR updated with its base branch as soon as it has 1
```
Comment (Kyle-Verhoog) [inline .mergify.yml:11]: 👍

**Review by Kyle-Verhoog — APPROVED** [review]
Now if only we could get our CI pass rate up so that mergify can merge our PRs 😭

**Comment by Kyle-Verhoog** [comment]
image
🤔

**Comment by brettlangdon** [comment]
> image
> 🤔

This is because the first commit from @jd isn't signed. But since we do squash merges that end up being signed, it is ok. It'll just block manually merging via the UI.

**Pull request merged**